opm/parser/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -72,14 +72,21 @@ namespace Opm {
     public:
         explicit EclipseGrid(const std::string& filename);
         explicit EclipseGrid(const EclipseGrid& srcGrid);
-        explicit EclipseGrid(size_t nx, size_t ny, size_t nz,
-                             double dx = 1.0, double dy = 1.0, double dz = 1.0);
+	EclipseGrid(size_t nx, size_t ny, size_t nz,
+		    double dx = 1.0, double dy = 1.0, double dz = 1.0);
+	
+	EclipseGrid(const std::vector<int>& dims , 
```

Comment:
> Could we have a very short comment on the content of mapaxes?

MAPAXES is an Eclipse input keyword which can be used to rotate and move the grid to real-world location, i.e. UTM coordinates.